spec/models/pawn_spec.rb

```diff
@@ -1,5 +1,34 @@
 require 'rails_helper'
 
 RSpec.describe Pawn, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it 'can move vertically and two squares on first move' do
+    game = Game.create
+    pawn = Pawn.create(game_id: game.id, x_position: 3, y_position: 1, color: 'white')
+
+    expect(pawn.first_move?(2)).to be true
```

Comment:
In your tests, you probably don't need to go as detailed as testing the specific sub-methods that make up `valid_move?` - you should just be able to use `pawn.valid_move?` for all of these tests.  👍